tests/test_filtsmooth/test_statespace/test_integrator.py

```diff
@@ -57,7 +57,7 @@ def test_proj2coord(self):
 
 class TestIBM(unittest.TestCase, NumpyAssertions):
     def setUp(self):
-        self.sde = pnfs.statespace.IBM(ordint=2, spatialdim=2, diffconst=DIFFCONST)
+        self.sde = pnfs.statespace.IBM(ordint=2, spatialdim=2)
```

Comment:
In all these tests I basically changed the same stuff: I removed the diffusion from the constructor, and instead added the same diffusion into all `transition` functions that are called in the tests